### PR TITLE
fix: browserify build/test flakes

### PR DESCRIPTION
**Pull request by joshgummersall**
- Clear out browser-related files in bf-connector (cause of dotjs test flakes)
- More consistent use of "browser" setting, exorcist, etc.

cc @BruceHaley

### CODE REVIEW

File: libraries/botframework-connector/package.json [hunk 1]
```diff
@@ -50,7 +50,8 @@
   },
   "scripts": {
     "build": "tsc",
-    "postbuild": "shx cp lib/index.js lib/browser.js | browserify -s BFC --debug lib/browser.js | exorcist lib/browser.js.map | sponge lib/browser.js",
+    "browserify": "browserify -s BFC --debug lib/browser.js | exorcist lib/browser.js.map | sponge lib/browser.js",
+    "postbuild": "rimraf lib/browser.* && shx cp lib/index.js lib/browser.js && npm run browserify",
```
Comment (joshgummersall) [inline libraries/botframework-connector/package.json:54]: Test flakes were caused by `lib/browser.js` and/or `lib/browser.js.map` files from previous builds. Cleaning them up before re-browserifying fixes that.

File: libraries/botframework-streaming/package.json [hunk 2]
```diff
@@ -39,15 +39,14 @@
...
-    "sponge": "^0.1.0",
     "ts-node": "^4.1.0",
     "typescript": "3.5.3"
   },
   "scripts": {
     "build": "npm-run-all -p build:lib build:es5",
     "build:lib": "tsc -p tsconfig.json",
     "build:es5": "tsc -p tsconfig-es5.json",
-    "postbuild": "browserify -s BFSE --debug es5/index-browser.js | exorcist lib/index-browser.js.map | sponge lib/index-browser.js",
+    "postbuild": "browserify -s BFSE --debug es5/index-browser.js | exorcist lib/index-browser.js.map > lib/index-browser.js",
```
Comment (joshgummersall) [inline libraries/botframework-streaming/package.json:49]: `sponge` isn't necessary as we aren't processing a file in-place.

**Review by stevengum — APPROVED** [review]
LGTM, are we going to need this in 4.11?

**Comment by joshgummersall** [comment]
@stevengum I don't think so - the only "observable" difference to consumers should be the `adaptive-expressions` `"browser"` setting. I don't think they use or care about that, though. If we cut another `4.11.x` release, I suppose we should include this, but I wouldn't block the RCs for it.

**Pull request merged**